docs/en_US/reference/experiment_config.rst

```diff
@@ -538,10 +538,10 @@ If `trialGpuNumber`_ is less than the length of this value, only a subset will b
 This will be used as ``CUDA_VISIBLE_DEVICES`` environment variable.
 
 
-trialPrepareCommand
+trialPythonPath
```

Comment:
what is the difference between `pythonPath` and `trialPythonPath`?

Reply:
pythonPath is config v1
trialPythonPath is config v2

Reply:
why use different names?

Reply:
Just follow the previous naming way: `preCommand` is config v1, `trialPrepareCommand` is config v2. Or use `pythonPath` is enough? @liuzhe-lz

Reply:
I'm not sure which one is better.
My point is that it must be clear whether the path is a local path or a path on remote server, or a path in docker container.

Reply:
could someone explain when to use `pythonPath` and when `trialPythonPath` should be used?